### PR TITLE
Add README for drawer

### PR DESCRIPTION
**Pull request by azakus**
Related #348

### CODE REVIEW

File: packages/drawer/README.md [hunk 1]
```diff
@@ -1,64 +1,143 @@
...
+```html
+<mwc-drawer>
+    <div>
+        <p>Drawer Content!</p>
+    </div>
+    <div slot="appContent">
+        <mwc-top-app-bar>
+            <div slot="title">Title</div>
+        </mwc-top-app-bar>
+    <div>
```
Comment (aomarks) [inline packages/drawer/README.md]: Unbalanced tag here and below

File: packages/drawer/README.md [hunk 2]
```diff
@@ -1,64 +1,143 @@
...
+`appContent` | Elements to display in the "app content" to the right of, or under, the drawer.
+*default* | Elements to display under the header in drawer.
+
+
+### Properties/Attributes
+
+Name | Type | Default | Description
+---- | ---- | ------- | -----------
+`open` | `boolean` | `false` | Whether the dialog is open
+`hasHeader` | `boolean` | `false` | When `true`, adds
```
Comment (aomarks) [inline packages/drawer/README.md]: missing end of sentence

File: packages/drawer/README.md (hunk 1 above)
Comment (azakus) [reply packages/drawer/README.md]: Done

File: packages/drawer/README.md [hunk 3]
```diff
@@ -1,64 +1,143 @@
...
+<img src="images/dismissable.gif" width="552px">
+
+```html
+<mwc-drawer hasHeader type="dismissable">
+    <span slot="title">Drawer Title</span>
+    <span slot="subtitle">subtitle</span>
+    <div>
+        <p>Drawer content!</p>
+        <mwc-icon-button icon="gesture"></mwc-icon-button>
+        <mwc-icon-button icon="gavel" id="gavel"></mwc-icon-button>
```
[the remark this answers is not in the document]
Comment (azakus) [reply packages/drawer/README.md]: Done

File: packages/drawer/README.md (hunk 2 above)
Comment (azakus) [reply packages/drawer/README.md]: Done.

**Pull request merged**